src/js/utils.js

```diff
@@ -57,7 +57,9 @@ export function iconPad(props) {
 
   let style = '';
   if (height && theme?.text?.[height]?.height) {
-    const lineHeight = parseMetricToNum(theme.text[height].height);
+    const unit = theme.text[height].height.match(/[a-z]+$/)?.[0] || 'px';
+    const lineHeight =
+      parseMetricToNum(theme.text[height].height) * (unit === 'rem' ? 16 : 1);
```

Comment:
Where does the 16 come from? Assuming this assumes and 16px root? Can we document this with a comment and/or a static variable (e.g. `const ROOT_UNIT = 16`)?